doc/api.md

```diff
@@ -487,17 +883,17 @@ translation from geo:point to geo:json)
         {
           "name":"location",
           "type":"geo:point",
-          "expression": "${@latitude}, ${@longitude}",
+          "expression": "latitude, longitude",
           "reverse": [
             {
               "object_id":"longitude",
-              "type": "Text",
-              "expression": "${trim(substr(@location, indexOf(@location, \",\") + 1, length(@location)))}"
+              "type": "Number",
+              "expression": "location | split(', ')[0] | parsefloat()"
```

Comment:
Looking this kind of changes... should we understand that all the mentions to the old legacy expression legacy has been removed from documentation?

Reply:
For documents processed yes. This should be checked again in dedicated PR https://github.com/telefonicaid/iotagent-node-lib/pull/1339

Reply:
NTC